content/content-strategy/identify-business-needs/what-business-needs/index.yml

```diff
@@ -0,0 +1,27 @@
+pagetitle: What business needs to look for
+description: You’ll hear a wide range of business objectives. Know what you’re looking for before you begin.
+weight: 10
```

Comment:
Compare with all other weight values in the parent topic and Content strategy module. Will cause list order issues in the nav.

Reply:
Fixed